omero_figure/templates/figure/index.html

```diff
@@ -596,14 +596,14 @@ <h4>Dynamic properties</h4>
                             <td>Time: -0.06 s</td>
                         </tr>
                         <tr>
-                            <th>Z slice with units position <br/>relative to the 3rd frame</th>
+                            <th>Z index with units <br/> (first plane is Z: 0.00 &#181;m), <br/>offset by 3 times the Z pixel size</th>
                             <td>Z: [z.unit; offset=3]</td>
                             <td>Z: -3.00 &#181;m </td>
                         </tr>
                         <tr>
-                            <th>Z slice with index position <br/>relative to the 3rd frame</th>
+                            <th>Z index with (first plane is Z: 1) <br/>offset by 3</th>
```

Comment:
I think we don't need the `with` in this case? Otherwise looks good.

![Screenshot 2023-11-20 at 13 03 56](https://github.com/ome/omero-figure/assets/900055/a35a8dc3-7f57-42c0-b7f2-ef0971644760)

Reply:
True, thanks. !
This is a typo.